lib/msf/core/payload/custom.rb

```diff
@@ -0,0 +1,9 @@
+# -*- coding => binary -*-
+
+#
+module Msf::Payload::Custom
+
+  def dynamic_stage_size?
```

Comment:
Let's rename this to be more specific, so it doesn't overlap/cause confusion with these methods:

https://github.com/rapid7/metasploit-framework/blob/82daa0c90f915cc3b4fc47bb5c8d733544d02582/lib/msf/core/payload.rb#L157-L171